source/applications/advanced/gamma_correction.py

```diff
@@ -59,7 +59,7 @@ def _capture_2d_image(camera):
     settings_2d = zivid.Settings2D(acquisitions=[zivid.Settings2D.Acquisition()],)
     with camera.capture(settings_2d) as frame_2d:
         image = frame_2d.image()
```

Comment:
Won't run. This is "frame_2d.image_rgba()" now.

Reply:
done.